### PR TITLE
[React] Move Solana to peer dependencies

### PR DESCRIPTION
**Pull request by joaquim-verges**

### CODE REVIEW

File: .changeset/lovely-hotels-bake.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/react": patch
```
Comment (jnsdls) [inline .changeset/lovely-hotels-bake.md:2]: ur also changing things in sdk so missing a changeset for that

**Review by jnsdls — CHANGES_REQUESTED** [review]

File: .changeset/lovely-hotels-bake.md (hunk 1 above)
Comment (joaquim-verges) [reply .changeset/lovely-hotels-bake.md:2]: done

**Pull request merged**